src/config/constants/types.ts

```diff
@@ -10,6 +10,7 @@ export interface Token {
   address?: Address
   decimals?: number
   projectLink?: string
+  price?: string
```

Comment:
At the moment I'm attaching the token price to the `token` within farm state, i.e.

`farm.quoteToken.price`, `farm.token.price`.

The alternative to this would be to just sit it at the `farm` level:

`farm.quoteTokenPriceVsBusd`, `farm.tokenPriceVsBusd`